src/translations/et/common.json

```diff
@@ -71,13 +74,5 @@
   "widget.action.confirm": "Kinnita",
   "widget.action.download-chat": "Lae vestlus alla",
   "widget.action.forward-chat": "Edasta vestlus e-mailile",
-  "widget.action.forward-email": "Teie e-postiaadress",
```

Comment:
Please revert waiting time removed translations as it they will be referenced here https://github.com/buerokratt/Buerokratt-Chatbot/issues/445